dispatcher/dispatcher.go

```diff
@@ -209,12 +211,12 @@ func (d *IotxDispatcher) EventAudit() map[iotexrpc.MessageType]int {
 }
 
 func (d *IotxDispatcher) actionHandler() {
+	d.wg.Done()
```

Comment:
if move to here, `Stop()` can exit while some `actionHandlers()` go-routine are still running, which is not intended?

Reply:
thx. it's a typo